docs/reference/process.md

```diff
@@ -62,8 +62,12 @@ The following task properties are defined in the process body:
 :::{versionadded} 25.10.0
 :::
 
+:::{warning}
+This is a preview feature – the syntax and behavior may change in future releases.
+:::
+
 :::{note}
-Typed processes require the `nextflow.preview.types` feature flag to be enabled in every script that uses them.
+This feature requires the `nextflow.preview.types` feature flag to be enabled in every script that uses typed processes.
```

Comment:
Here also I think the current language was preferred by Chris

Reply:
```suggestion
This is a preview feature and requires the `nextflow.preview.types` feature flag to be enabled in every script that uses typed processes. The syntax and behavior may change in future releases.
```